docs/workflow_generators/residential_hpxml.rst

```diff
@@ -91,13 +94,18 @@ Arguments
   - ``measure_dir_name``: Name of measure directory.
   - ``arguments``: map of key, value arguments to pass to the measure.
 
-- ``server_directory_cleanup`` (optional): optionally preserve or delete
+- ``server_directory_cleanup`` (optional): Optionally preserve or delete
   various simulation output files. These arguments are passed directly to
   the `ServerDirectoryCleanup`_ measure in resstock. Please refer to the
   measure arguments there to determine what to set them to in your config file.
   Note that the default behavior is to retain some files and remove others.
   See :ref:`server-dir-cleanup-defaults` for current defaults.
 
+- ``debug`` (optional): Optionally enable debug mode. Enabling debug
+  mode will preserve all simulation input and output files, including but
+  not limited to: in.osm, all EnergyPlus output files, and intermediate
+  existing and upgraded files (e.g., OSWs and XMLs).
```

Comment:
Documenting the new top level debug argument that can optionally be called out in the yml file.